azurerm/internal/services/logic/logic_app_workflow_resource.go

```diff
@@ -290,6 +303,10 @@ func resourceArmLogicAppWorkflowRead(d *schema.ResourceData, meta interface{}) e
 			}
 		}
 
+		if props.IntegrationServiceEnvironment != nil && props.IntegrationServiceEnvironment.ID != nil {
+			d.Set("integration_service_environment_id", props.IntegrationServiceEnvironment.ID)
+		}
+
 		if props.IntegrationAccount != nil && props.IntegrationAccount.ID != nil {
 			d.Set("logic_app_integration_account_id", props.IntegrationAccount.ID)
 		}
```

Comment:
I realise this isn't your change - but as we're changing the item above, could we fix this too:

```suggestion
		integrationAccountId := ""
		if props.IntegrationAccount != nil && props.IntegrationAccount.ID != nil {
 			integrationAccountId := *props.IntegrationAccount.ID
 		}
 		d.Set("logic_app_integration_account_id", integrationAccountId)
```